code/nx.json

```
@@ -58,12 +58,19 @@
       "cache": true
     },
     "check": {
+      "executor": "nx:run-commands",
+      "options": {
+        "cwd": "{projectRoot}",
+        "command": "yarn typecheck"
```

Could we keep this name `check`, to reduce the amount of changes needed?
```suggestion
        "command": "yarn check"
```

---

See above, as far as I can tell nx will prefer package.json definitions to its own if the names match. In the likely event I'm wrong it can be changed, but it took me five times as long to get nx working as to get the type checking working.